canonical-web-and-design/snapcraft.io
# Releases accordion

Fixes: https://github.com/canonical-web-and-design/snapcraft.io/issues/2371
Fixes: https://github.com/canonical-web-and-design/snapcraft.io/issues/2373
Fixes: https://github.com/canonical-web-and-design/snapcraft.io/issues/2372

## QA
- Pull branch
- `./run`
- Visit http://0.0.0.0:8004/snap_name/releases
- Use the release UI as normal
- Things that won't work properly - updating and cancelling progressive releases
- Everything else should be ok (or have bugs)

## Screenshots
### Closed
![Screenshot_2019-12-03  Releases and revision history for lukewh-test (3)](https://user-images.githubusercontent.com/479384/70060372-54eb9300-15da-11ea-8df2-8a56e6991c8d.png)
### Open
New progressive release, progressive release % update, release to a a channel+arch with no previous release, closing channel
![Screenshot_2019-12-03  Releases and revision history for lukewh-test (4)](https://user-images.githubusercontent.com/479384/70060373-55842980-15da-11ea-9fa9-99d7e07d8723.png)

New progressive release, progressive release pause, release to a a channel+arch with no previous release, closing channel
![Screenshot_2019-12-03  Releases and revision history for lukewh-test (5)](https://user-images.githubusercontent.com/479384/70060374-55842980-15da-11ea-9196-768aa3a7754e.png)

New progressive release, progressive release cancel, release to a a channel+arch with no previous release, closing channel
![Screenshot_2019-12-03  Releases and revision history for lukewh-test (6)](https://user-images.githubusercontent.com/479384/70060376-561cc000-15da-11ea-96a1-5b2a561d644e.png)


TODO:
- [x] Move setting of progressive key on release
